Software/wittypi/daemon.sh

```diff
@@ -152,10 +152,34 @@ while true; do
       clear_alarm_flags
     else
     	# not shutdown by alarm
+      # wait for button to be pressed for SHUTDOWN_HOLD_TIME to prevent single accidental button shutdown
+      # by default this is set to 2 seconds, can be changed in utilities.sh
+      counter=0
+      while [ $counter -lt $SHUTDOWN_HOLD_TIME ]; do  # increase this value if it needs more time
+        if [ $(gpio -g read $HALT_PIN) == '1' ] ; then
```

Comment:
Sure, that the `1` will be correct?
After my Debug-Session of the correct value: `while true; do gpio -g read 4; sleep 1; done` I will get a `0` back, if I press the button. The default value of the $HALT_PIN is `1`

Reply:
I test my changes on Raspbian OS (Buster, Bullseye) and now it works.
[Please replace the 1 with a 0.](https://github.com/lgremme/Witty-Pi-3/blob/975566215c154073c4e2cb124e1a0a07dc228c3a/data/usr/share/wittypi/wittypi-daemon#L159)

For a quick check, you can ask the GPIO-PIN 4:
`while true; do gpio -g read 4; sleep 1; done`